plug-ins/clan/impl/battlerager.cpp

```diff
@@ -485,6 +485,12 @@ SKILL_RUNP( spellbane )
 
         affect_to_char(ch,&af);
 
+        af.modifier = 0;
```

Comment:
@Kerrad affects can have both location/modifier and bitvector filled in independently. See `haste` spell as an example. That way it will only take 1 line in `affects` output, rather than 2.

Reply:
done, thx for input